Week3/functions.js

```diff
@@ -0,0 +1,268 @@
+const { execQuery } = require('./database');
+const { isInputValid } = require('./validate');
+
+async function showRecords(path) {
+  try {
+    const table = path.substring(1);
+    return await execQuery(`SELECT * FROM ${table}`);
+  } catch (error) {
+    return error.message;
+  }
+}
+
+async function showRecordById(path, id) {
```

Comment:
You are totally right..In first time I defined so many function for each endpoints to show records.. I noticed that I repeated same things for each function and then I tried to write clean code and shouldn't repeat and so I wrote like so.. Actually I thought that it might be unsafe.. In the future I am not gonna do that again .. Okay..thanks